src/MSBuild.Sdk.SqlProj/MSBuild.Sdk.SqlProj.nuspec

```diff
@@ -14,10 +14,12 @@
       <packageType name="$packagetype$" />
     </packageTypes>
     <repository type="git" url="https://github.com/rr-wfm/MSBuild.Sdk.SqlProj/" />
+    <readme>docs\README.md</readme>
   </metadata>
   <files>
     <file src="Sdk\**" target="Sdk" />
     <file src="tools\**" target="tools" />
     <file src="LICENSE.txt" target="LICENSE.txt" />
+    <file src="..\..\README.md" target="docs\" />
```

Comment:
Should it not just be: ..\README.md ?

Reply:
Mmm... I was just wondering where the `LICENCE.txt` comes from. A target (SetNuSpecProperties) in the project file copies it over from the solution root. I could do the same trick for the readme.
Just wondering if there is an added benefit?

Reply:
The solution root is two levels up from the project file, so `..\..\README.md` is correct. However, I'ld like to keep it in line with what is already there. So I'll copy it to the nuspec location first, just as the `LICENSE` file..

Reply:
On the other hand... this might lead to inadvertently modifying the wrong README.md.
Maybe it's better for the README.md to not copy it?

Reply:
Just leave it as is, I think

Reply:
Sorry, which version do you mean? The first, so `..\..\README.md` or the second one with the copy action?

Reply:
Original

Reply:
If I remember correctly the reason the LICENSE.txt is copied first is because the original file is called just LICENSE which NuGet cannot deal with (a file without an extension) so that's why we copy it as LICENSE.txt so that it can be packaged.